app/release-notes/release-notes.tsx

```diff
@@ -0,0 +1,45 @@
+import {getVersion} from "app/core/utils/get-version"
+import React from "react"
+import ReactMarkdown from "react-markdown"
+import Link from "src/js/components/common/Link"
+import styled from "styled-components"
+import {useReleaseNotes} from "./use-release-notes"
+
+const Scrollable = styled.div`
+  overflow: auto;
+`
+const BG = styled.div`
+  margin: 48px auto;
+  padding: 24px;
+  max-width: 600px;
+  line-height: 1.6;
+  a {
+    color: var(--azure);
+  }
+  hr {
+    border: none;
+    border-top: 1px solid var(--cloudy);
+  }
+`
+
+const components = {
+  // eslint-disable-next-line
+  a: ({children, ...props}) => {
+    return <Link {...props}>{children}</Link>
+  }
+}
+
+const version = getVersion()
+
+export default function ReleaseNotes() {
+  const notes = useReleaseNotes(version)
+
+  return (
+    <Scrollable>
+      <BG className="release-notes">
```

Comment:
I can't find where you're using this class (and am hoping that it is simply unused so that all of this component's styling is in one place 😬 ). Do we need it?

Reply:
Nope, that was used when I was developing, but it's not needed anymore. Thanks!